.gitignore

```diff
@@ -0,0 +1,4 @@
+Package.resolved
```

Comment:
I think we should check this in so we have a record in source control of what versions of dependencies we're using.

Reply:
Sure